org.eclipse.lsp4xml.extensions.maven/src/main/java/org/eclipse/lsp4xml/extensions/maven/MavenProjectCache.java

```diff
@@ -95,7 +97,7 @@ private void parse(DOMDocument document) {
 			}
 			File file = new File(uri);
 			workingCopy = File.createTempFile("workingCopy", '.' + file.getName(), file.getParentFile());
-			Files.copy(file.toPath(), workingCopy.toPath(), StandardCopyOption.REPLACE_EXISTING);
+			Files.writeString(workingCopy.toPath(), document.getText(), StandardOpenOption.CREATE);
```

Comment:
Files.writeString() was introduced in Java 11, and the project uses Java 8. Maybe use [Files.write](https://kodejava.org/how-do-i-write-a-text-file-in-jdk-7/)?